### PR TITLE
Remove unnecessary backslashes from translations

### PR DESCRIPTION
**Pull request by nickvergessen**
@DeepDiver1975 

(Btw where is this used, neither the occ, nor the UI seems to use that code?!)

### CODE REVIEW

File: lib/private/app.php [hunk 1]
```diff
@@ -1118,7 +1118,7 @@ public static function installApp($app) {
 			if(!empty($missing)) {
 				$missingMsg = join(PHP_EOL, $missing);
 				throw new \Exception(
-					$l->t('App \"%s\" cannot be installed because the following dependencies are not fulfilled: %s',
+					$l->t('App "%s" can\'t be installed because the following dependencies are not fulfilled: %s',
```
Comment (DeepDiver1975) [inline lib/private/app.php]: please keep cannot

File: lib/private/app.php [hunk 2]
```diff
@@ -1106,7 +1106,7 @@ public static function installApp($app) {
 			$version=OC_Util::getVersion();
 			if(!self::isAppCompatible($version, $info)) {
 				throw new \Exception(
-					$l->t('App \"%s\" can\'t be installed because it is not compatible with this version of ownCloud.',
+					$l->t('App "%s" can\'t be installed because it is not compatible with this version of ownCloud.',
```
Comment (DeepDiver1975) [inline lib/private/app.php]: should be cannot as well

**Comment by DeepDiver1975** [comment]
> Btw where is this used, neither the occ, nor the UI seems to use that code?!)

this is the code which is executed when an app is installed via the apps management

**Comment by PVince81** [comment]
Makes sense :+1:

**Comment by MorrisJobke** [comment]
:+1:

**Pull request merged**